ind.ie.Gnomit.appdata.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>ind.ie.Gnomit.desktop</id>
```

Comment:
Shouldn't this `<id>` element's value match `app-id` in `ind.ie.Gnomit.json`? [This](https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html) example, while slightly different, refers to the `<launchable>` tag for what you have here.

> Example: If your application's `.desktop` file is named `org.example.FooBar.desktop`, a good `component-id` would be `org.example.FooBar`.

Reply:
Just updated and pushed based on your suggestions.

I believe the entry in there was what Gnome Builder had auto-generated.